### PR TITLE
CORTX-32072: Updated badges and solutions sections with new links

### PR DESCRIPTION
**Pull request by swatid-seagate**
Signed-off-by: Swatid <swati.dhotarkar@seagate.com>

# Problem Statement
- Updated README.md for new badges and new links added in solutions section.

# Documentation
  Checklist for Author
- [x] Changes done to README.md

-----
[View rendered README.md](https://github.com/swatid-seagate/cortx-re/blob/re-solutions-update/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
-[![Codacy Badge](https://app.codacy.com/project/badge/Grade/edb2670e6aa24aeb899c496c15b596c9)](https://www.codacy.com?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Seagate/cortx-re&amp;utm_campaign=Badge_Grade) [![License: AGPL v3](https://img.shields.io/badge/License-AGPL%20v3-blue.svg)](https://github.com/Seagate/cortx-re/blob/main/LICENSE)
-[![YouTube Badge](https://img.shields.io/badge/Video-YouTube-red)](https://cortx.link/videos )
+[![Codacy Badge](https://app.codacy.com/project/badge/Grade/a3d60ecc5d8942c9a4b04bcf4b60bf20)](https://www.codacy.com/gh/Seagate/cortx/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Seagate/cortx&amp;utm_campaign=Badge_Grade)
```
Comment (shailesh-vaidya) [inline README.md:1]: Codacy badge should go to `https://app.codacy.com/gh/Seagate/cortx-re/dashboard`  currently it's going to  `https://app.codacy.com/organizations/gh/Seagate/repositories`

File: README.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+[![license](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/Seagate/cortx/blob/main/LICENSE)
 [![Slack](https://img.shields.io/badge/chat-on%20Slack-blue")](https://cortx.link/join-slack)
-[![GitHub contributors](https://img.shields.io/github/contributors/Seagate/cortx-re)](https://github.com/Seagate/cortx-re/graphs/contributors/)
+[![YouTube](https://img.shields.io/badge/Video-YouTube-red)](https://cortx.link/videos)
+[![Latest Release](https://img.shields.io/github/v/release/Seagate/cortx?label=Latest%20Release)](https://github.com/seagate/cortx/releases/latest)
+[![GitHub contributors](https://img.shields.io/github/contributors/Seagate/cortx)](https://github.com/Seagate/cortx/graphs/contributors/)
```
Comment (shailesh-vaidya) [inline README.md:6]: Contributor page should go to `cortx-re` contributors. Currently, it's going to `cortx` contributors.

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]
Please use `cortx-re` repo for all the badges.

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]
Please update the PR template. Add Jira ticket in the PR title.

**Comment by swatid-seagate** [comment]
> Please update the PR template. Add Jira ticket in the PR title.

It's done

File: README.md [hunk 3]
```diff
@@ -1,7 +1,10 @@
-[![Codacy Badge](https://app.codacy.com/project/badge/Grade/edb2670e6aa24aeb899c496c15b596c9)](https://www.codacy.com?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Seagate/cortx-re&amp;utm_campaign=Badge_Grade) [![License: AGPL v3](https://img.shields.io/badge/License-AGPL%20v3-blue.svg)](https://github.com/Seagate/cortx-re/blob/main/LICENSE)
-[![YouTube Badge](https://img.shields.io/badge/Video-YouTube-red)](https://cortx.link/videos )
+[![Codacy Badge](https://app.codacy.com/project/badge/Grade/edb2670e6aa24aeb899c496c15b596c9)](https://www.codacy.com/gh/Seagate/cortx-re/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Seagate/cortx-re&amp;utm_campaign=Badge_Grade)
+[![license](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/Seagate/cortx/blob/main/LICENSE)
```
Comment (shailesh-vaidya) [inline README.md:2]: cortx-re has `Affero General Public License v3.0`  Refer - https://github.com/swatid-seagate/cortx-re/blob/main/LICENSE
Badge is going to https://github.com/Seagate/cortx/blob/main/LICENSE

File: README.md [hunk 4]
```diff
@@ -1,7 +1,10 @@
...
 [![Slack](https://img.shields.io/badge/chat-on%20Slack-blue")](https://cortx.link/join-slack)
+[![YouTube](https://img.shields.io/badge/Video-YouTube-red)](https://cortx.link/videos)
+[![Latest Release](https://img.shields.io/github/v/release/Seagate/cortx?label=Latest%20Release)](https://github.com/seagate/cortx-re/releases/latest)
 [![GitHub contributors](https://img.shields.io/github/contributors/Seagate/cortx-re)](https://github.com/Seagate/cortx-re/graphs/contributors/)
+[![SODA Eco project](https://img.shields.io/badge/SODA-ECO%20Project-9cf)](./doc/Soda-welcome-page.md)
```
Comment (shailesh-vaidya) [inline README.md:7]: SODA Project link is giving a `404 ` error 

image

File: README.md [hunk 5]
```diff
@@ -1,7 +1,10 @@
-[![Codacy Badge](https://app.codacy.com/project/badge/Grade/edb2670e6aa24aeb899c496c15b596c9)](https://www.codacy.com?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Seagate/cortx-re&amp;utm_campaign=Badge_Grade) [![License: AGPL v3](https://img.shields.io/badge/License-AGPL%20v3-blue.svg)](https://github.com/Seagate/cortx-re/blob/main/LICENSE)
-[![YouTube Badge](https://img.shields.io/badge/Video-YouTube-red)](https://cortx.link/videos )
+[![Codacy Badge](https://app.codacy.com/project/badge/Grade/edb2670e6aa24aeb899c496c15b596c9)](https://www.codacy.com/gh/Seagate/cortx-re/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Seagate/cortx-re&amp;utm_campaign=Badge_Grade)
+[![license](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/Seagate/cortx/blob/main/LICENSE)
 [![Slack](https://img.shields.io/badge/chat-on%20Slack-blue")](https://cortx.link/join-slack)
+[![YouTube](https://img.shields.io/badge/Video-YouTube-red)](https://cortx.link/videos)
+[![Latest Release](https://img.shields.io/github/v/release/Seagate/cortx?label=Latest%20Release)](https://github.com/seagate/cortx-re/releases/latest)
```
Comment (shailesh-vaidya) [inline README.md:5]: This should go to `cortx` repo as there is no release in `cortx-re`

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (swatid-seagate) [reply README.md:5]: ok. Done.

**Comment by shailesh-vaidya** [comment]
Closing this PR

**Pull request closed without merging**